dts/arm/st/h7/stm32h743.dtsi

```diff
@@ -14,6 +14,20 @@
 				erase-block-size = <DT_SIZE_K(128)>;
 			};
 		};
+
+		usbotg_fs: usb@40080000 {
+			compatible = "st,stm32-otghs";
```

Comment:
```suggestion
			compatible = "st,stm32-otgfs";
```

Reply:
Changed this and the interrupt name to use `fs` instead of `hs`.